src/elisp/treemacs-mouse-interface.el

```diff
@@ -277,9 +277,27 @@ and ignore any prefix argument."
                (list (xref-item-buffer xref) (xref-item-position xref)))))
           ('issue-warning
            (treemacs-log-failure "Tag '%s' is located in a buffer that does not exist."
-                          (propertize (treemacs-with-button-buffer btn (treemacs--get-label-of btn)) 'face 'treemacs-tags-face)))
+             (propertize (treemacs-with-button-buffer btn (treemacs--get-label-of btn)) 'face 'treemacs-tags-face)))
           (_ (error "[Treemacs] '%s' is an invalid value for treemacs-goto-tag-strategy" treemacs-goto-tag-strategy)))))))
 
+(defun treemacs--paste-point-to-minibuffer ()
```

Comment:
This command is public, so its name needs to have only one dash.

It is also not strictly part of the mouse interface, but is a more general utility, so it should be part of treemacs-interface.el. 

The name should also change to paste-path-at-point-to-minibuffer so it's clear what exactly it does.